src/main/java/io/confluent/connect/jdbc/util/DateTimeUtils.java

```diff
@@ -141,10 +142,13 @@ public static String toEpochMicrosString(Timestamp timestamp) {
    * @return the epoch nanoseconds string
    */
   public static String toEpochNanosString(Timestamp timestamp) {
-    return Optional.ofNullable(timestamp)
-        .map(DateTimeUtils::convertToEpochNanos)
-        .map(String::valueOf)
-        .orElse(null);
+    if (timestamp == null) {
```

Comment:
nit: We can keep the method chaining style as it was before.
```
public static String toEpochNanosString(Timestamp timestamp) {
    return Optional.ofNullable(timestamp)
        .map(DateTimeUtils::convertToEpochNanosBig)
        .map(BigInteger::toString)
        .orElse(null);
}
```
And keep the computation inside `convertToEpochNanosBig` method in DateTimeUtils.